tests/contrib/aiohttp/conftest.py

```diff
@@ -3,7 +3,8 @@
 import pytest
 
 from ddtrace.contrib.aiohttp.middlewares import trace_app
-from ddtrace.contrib.aiohttp.patch import patch
+from ddtrace.contrib.aiohttp.patch import patch as patch_aiohttp
+from ddtrace.contrib.aiohttp_jinja2 import patch as patch_aiohttp_jinja2
```

Comment:
do we want this here? seem like we can remove?

Reply:
Patching aiohttp_jinja2? We still want to do it since the test suite is used for both integrations